src/test/java/seedu/address/logic/commands/DurationCommandTest.java

```diff
@@ -0,0 +1,134 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_DURATION_AMY;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_DURATION_BOB;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandFailure;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.logic.commands.CommandTestUtil.showPersonAtIndex;
+import static seedu.address.testutil.HomeworkTestTypicalPersons.getTypicalAddressBook;
```

Comment:
`HomeworkTestTypicalPersons` has been deleted already.

Reply:
Hi ms joanna ya sorry still work in progress, please hang on :")